XpringKit/XRP/DefaultXRPClient.swift

```diff
@@ -198,6 +201,11 @@ extension DefaultXRPClient: XRPClientDecorator {
 
     let request = Org_Xrpl_Rpc_V1_GetTransactionRequest.with {
       $0.hash = transactionHashData
+      $0.ledgerRange = Org_Xrpl_Rpc_V1_LedgerRange.with {
```

Comment:
I validated with Mark Travis on rippled that we don't ned to set this. If unset, all ledgers are searched. 

You can omit this code.